### PR TITLE
feat: add daily-challenge day-24/10

### PR DESCRIPTION
**Pull request by ntlem**
## Summary by CodeRabbit

- **New Features**
	- Introduced a function to convert Vietnamese musical notes to English note representations.
	- Added a sample array to demonstrate the conversion functionality.

### CODE REVIEW

File: daily-challenges/2024-10/24/ntlem.spec.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function convertNotes(notes: string[]): string[] {
+    const a: { [key: string]: string } = {
```
Comment (minhphong306) [inline daily-challenges/2024-10/24/ntlem.spec.ts:2]: Nên đặt một cái tên cụ thể hơn chút. 
VD: noteMap

File: daily-challenges/2024-10/24/ntlem.spec.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "rê": "D",
+        "mi": "E",
+        "fa": "F",
+        "sol": "G",
+        "la": "A",
+        "si": "B"
+    };
+
+    return notes.map(note => a[note] || note);
+}
```
[the remark this answers is not in the document]
Comment (minhphong306) [reply daily-challenges/2024-10/24/ntlem.spec.ts:13]: Nice idea. Thêm check vào @ntlem

**Review by minhphong306 — CHANGES_REQUESTED** [review]
Please fix comments.

File: daily-challenges/2024-10/24/ntlem.spec.ts (hunk 1 above)
Comment (ntlem) [reply daily-challenges/2024-10/24/ntlem.spec.ts:2]: fixed

File: daily-challenges/2024-10/24/ntlem.spec.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ntlem) [reply daily-challenges/2024-10/24/ntlem.spec.ts:13]: fixed

**Review by minhphong306 — APPROVED** [review]
LGTM

**Pull request merged**